components/Common/Select/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { randomBytes } from 'crypto';
+
+import { ChevronDownIcon } from '@heroicons/react/24/outline';
+import * as Primitive from '@radix-ui/react-select';
+import { useId } from 'react';
+import type { FC } from 'react';
+
+import styles from './index.module.css';
+
+type SelectProps = {
+  values: string[];
+  defaultValue?: string;
+  placeholder?: string;
+  dropdownLabel?: string;
+  label?: string;
+  onChange?: (value: string) => void;
+};
+
+const Select: FC<SelectProps> = ({
+  values = [],
+  defaultValue,
+  placeholder,
+  label,
+  dropdownLabel,
+  onChange,
+}) => {
+  const id = useId();
+  const key = randomBytes(16).toString('hex');
```

Comment:
```suggestion
  const componentKey = useMemo(() => randomBytes(16).toString('hex'), []);
```

Ensures uniqueness per component.

Reply:
Note that this is not needed at all.